src/Xamarin.Android.Build.Tasks/Tasks/PrependEnvironmentVariable.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Xamarin.Tools.Zip;
+using Microsoft.Android.Build.Tasks;
+
+namespace Xamarin.Android.Tasks
+{
+	public class PrependEnvironmentVariable : AndroidTask
+	{
+		public override string TaskPrefix => "PENV";
+
+		[Required]
+		public string Name { get; set; }
+
+		[Required]
+		public string Value { get; set; }
+
+		public override bool RunTask ()
+		{
+			string? oldValue = Environment.GetEnvironmentVariable (Name)?.Trim ();
+			string newValue;
+
+			if (!String.IsNullOrEmpty (oldValue)) {
+				newValue = $"{Value}:{oldValue}";
```

Comment:
This should be `;` on Windows, so you could use `System.IO.Path.PathSeparator` here.

Reply:
It's only used on Unix

Reply:
See here: https://github.com/xamarin/xamarin-android/pull/5971/files#diff-e2c60a31385c6fbadf9b9f439d3681ee357077bf3e66f181a0ddb02b9d98adf1R362-R376

Reply:
So we either need to make this task fail on Windows with an error code? Or just make it use the right separator?

Reply:
Look at the link :)

Reply:
Right, you have a `Condition`. But that doesn't prevent someone from trying to use this task on Windows one day?

It either should error on Windows if it isn't supported, or just make the 1 line change to support it.

Reply:
I'm going to remove this entire thing, it doesn't work anyway...